angrmanagement/ui/main_window.py

```diff
@@ -159,30 +167,68 @@ def _init_toolbars(self):
     def _init_menus(self):
         fileMenu = FileMenu(self)
         analyzeMenu = AnalyzeMenu(self)
+        viewMenu = ViewMenu(self)
         helpMenu = HelpMenu(self)
         self.menuBar().addMenu(fileMenu.qmenu())
         self.menuBar().addMenu(analyzeMenu.qmenu())
+        self.menuBar().addMenu(viewMenu.qmenu())
         self.menuBar().addMenu(helpMenu.qmenu())
 
     #
     # Workspace
     #
 
+    def _tabify(self):
+        self.right_dockable_views = [dock for dock in           self.workspace.dockable_views if dock.widget() is not None and dock.widget().default_docking_position == 'right']
+
+        for d0, d1 in zip(self.right_dockable_views, self.right_dockable_views[1:]):
+            self.central_widget.tabifyDockWidget(d0, d1)
+        self.right_dockable_views[0].raise_()
+
+
     def _init_workspace(self):
+        self.central_widget_main = QSplitter(Qt.Horizontal)
+        self.setCentralWidget(self.central_widget_main)
         self.central_widget = QMainWindow()
-        self.setCentralWidget(self.central_widget)
+        self.central_widget2 = QMainWindow()
+        self.central_widget_main.addWidget(self.central_widget)
+        self.central_widget_main.addWidget(self.central_widget2)
 
         wk = Workspace(self, Instance())
         self.workspace = wk
 
-        right_dockable_views = [dock for dock in self.workspace.dockable_views
-                                if dock.widget().default_docking_position == 'right']
+        self._tabify()
+        self.central_widget.setTabPosition(Qt.RightDockWidgetArea, QTabWidget.North)
 
-        for d0, d1 in zip(right_dockable_views, right_dockable_views[1:]):
-            self.central_widget.tabifyDockWidget(d0, d1)
-        right_dockable_views[0].raise_()
 
-        self.central_widget.setTabPosition(Qt.RightDockWidgetArea, QTabWidget.North)
+    def getCurrentTabId(self):
+        for i in range(1,6):
+            if self.right_dockable_views[i-1].visibleRegion().isEmpty() == False:
```

Comment:
I think `if not ...` is less misleading. Also, `getCurrentTabId` could just be a `current_tab` property... either way, prefer underscores over CamelCase.

Reply:
Please use `xxx is False` if you are comparing something with False.